diff --git a/node_modules/brace-expansion/index.js b/node_modules/brace-expansion/index.js
index bd19fe6..c4d9d97 100644
--- a/node_modules/brace-expansion/index.js
+++ b/node_modules/brace-expansion/index.js
@@ -153,6 +153,9 @@ function expand(str, isTop) {
    var incr = n.length == 3
      ? Math.abs(numeric(n[2]))
      : 1;
+    if (incr === 0) {
+      incr = 1;
+    }
     var test = lte;
     var reverse = y < x;
     if (reverse) {
