diff --git a/node_modules/react-native-svg/lib/module/web/WebShape.js b/node_modules/react-native-svg/lib/module/web/WebShape.js
index 594c0cc..5b3bf0f 100644
--- a/node_modules/react-native-svg/lib/module/web/WebShape.js
+++ b/node_modules/react-native-svg/lib/module/web/WebShape.js
@@ -5,8 +5,9 @@ unstable_createElement as createElement } from 'react-native';
 import { prepare } from './utils/prepare';
 import { convertInt32ColorToRGBA } from './utils/convertInt32Color';
 import { camelCaseToDashed, remeasure } from './utils';
-import { hasTouchableProperty } from './utils/hasProperty';
+import { hasTouchableProperty as _hasTouchableProperty } from './utils/hasProperty';
 import SvgTouchableMixin from '../lib/SvgTouchableMixin';
+const hasTouchableProperty = typeof _hasTouchableProperty === 'function' ? _hasTouchableProperty : (props => props.onPress || props.onPressIn || props.onPressOut || props.onLongPress);
 export class WebShape extends React.Component {
   prepareProps(props) {
     return props;
